lib/miq_expression/tag.rb

```diff
@@ -1,9 +1,10 @@
 class MiqExpression::Tag
   def self.parse(tag)
     klass, ns = tag.split(".")
+    klass, ns = "nil", klass if ns.nil? # support managed-label
```

Comment:
done - the tests are returning scary values.
Would like to change the code, but in another PR